### DISCUSSION TITLE
How to change the icon shown at taskbar of Windows

### DISCUSSION THREAD

**Discussion by quan787** [flet-dev/flet#378] [Q&A]
![image](https://user-images.githubusercontent.com/16741020/192043125-cedc365f-0704-4a67-9d3c-3a3c42452200.png)

also the "Flet description" in task manager

![image](https://user-images.githubusercontent.com/16741020/192043303-21f44826-701d-4916-a0e2-2feac7f315d1.png)

**Comment by FeodorFitsner** [comment]
The best way is to build your own "branded" or "white-label" Flet client. We will provide instructions and examples on how to do that.

**Reply by mortbauer** [reply]
I just started to use flet and it seems really awesome, but not being able to set an icon is kind of a blocker.

**Reply by mortbauer** [reply]
Now it got me hocked and did some investigations, am I right that in order to achieve this one has to compile an own version for the flet client (maybe also server) for every target and host it somewhere, at the same time the one would have to change the source of flet.py to use the custom assets? This seems like an incredible lot of work actually. 
Is this the way to go or is there some easier method on the way?

**Reply by QiCodeCN** [reply]
> The best way is to build your own "branded" or "white-label" Flet client. We will provide instructions and examples on how to do that.

I have the same question. Do you have an example to show how to do it? Thank you.

**Reply by FeodorFitsner** [reply]
Here's an update on the matter: https://github.com/flet-dev/flet/issues/113
